static/redfish/v1/schema/OemComputerSystem_v1.xml

```diff
@@ -65,6 +65,27 @@
                   <Annotation Term="OData.Description" String="List of enabled panel functions."/>
                   <Annotation Term="OData.LongDescription" String="This property shall contain the list of enabled panel functions."/>
                 </Property>
+                <Property Name="ChapData" Type="OemComputerSystem.ChapData">
+                  <Annotation Term="OData.Permissions" EnumMember="OData.Permission/Read"/>
+                  <Annotation Term="OData.Description" String="ChapData."/>
+                  <Annotation Term="OData.LongDescription" String="This property shall contain ChapData."/>
+                </Property>
+            </ComplexType>
+
+            <ComplexType Name="ChapData" BaseType="Resource.OemObject">
+                <Annotation Term="OData.AdditionalProperties" Bool="false"/>
+                <Annotation Term="OData.Description" String="ChapData."/>
+                <Annotation Term="OData.LongDescription" String="This type shall contain ChapData."/>
+                <Property Name="ChapName" Type="Edm.String">
+                  <Annotation Term="OData.Permissions" EnumMember="OData.Permission/Read"/>
+                  <Annotation Term="OData.Description" String="ChapName."/>
+                  <Annotation Term="OData.LongDescription" String="This property shall contain ChapName."/>
```

Comment:
Can you add the sentence from https://github.com/ibm-openbmc/phosphor-dbus-interfaces/blob/1060/yaml/com/ibm/PLDM/ChapData.interface.yaml#L11

Reply:
Updated